src/models/KeyModel.ts

```diff
@@ -60,7 +60,12 @@ export default class KeyModel {
     this.left = x * KEY_SIZE;
     this.top = y * KEY_SIZE;
 
-    this.color = c;
+    if (c === '#aaaaaa' || c === '#7777777') {
```

Comment:
```suggestion
    if (c === '#aaaaaa' || c === '#777777') {
```

@adamrocker Is `#777777` (6 digit)  the correct value? Also, the test code in the case of `#777777` should be added, if possible.